test/test_helper.rb

```diff
@@ -1,8 +1,11 @@
 # Configure Rails Environment
-ENV["RAILS_ENV"] = "test"
+ENV['RAILS_ENV'] = 'test'
 
-require File.expand_path("../dummy/config/environment.rb",  __FILE__)
-require "rails/test_help"
+require File.expand_path('../dummy/config/environment.rb',  __FILE__)
+require 'rails/test_help'
+
+require 'minitest/reporters'
+Minitest::Reporters.use!
```

Comment:
TravisCI seems to be failing because of this: https://travis-ci.org/devnacho/mountain_view/jobs/456272117

Would you mind removing it for this PR?

I'm happy to add it if you wanna open another PR for it :)